contracts/application/BaseApp.sol

```diff
@@ -88,17 +89,18 @@ contract BaseApp is Ownable {
   }
 
   /**
-   * @dev Gets the implementation for one of the owned proxies.
-   * @dev It's necessary to have this here because only the proxy owner can query it.
-   * @return the address of the current implemetation of the given proxy
+   * @dev Returns the current implementation of a proxy.
+   * This is needed because only the proxy owner can query it.
+   * @return The address of the current implementation of the proxy.
    */
   function getProxyImplementation(AdminUpgradeabilityProxy proxy) public view returns (address) {
     return proxy.implementation();
   }
 
   /**
-   * @dev Gets a proxy's admin. Necessary because only the admin can query it.
-   * @return the address of the current proxy admin of the given proxy
+   * @dev Returns the admin of a proxy.
+   * This is needed because only the admin can query it.
```

Comment:
"is needed because" is a bit strange in this case since this function might be superfluous (see discussion in https://github.com/zeppelinos/zos-lib/pull/36).

Maybe simply:
"Only the proxy admin can query it"?

Not feeling too strongly about this tho.

Reply:
Yours is better. It's short, but I don't think it requires more explanation. We will see if it causes confusion, and add more words later if so.